src/2024h2/candidates.md

```diff
@@ -30,19 +30,19 @@ The following goals are nearing completion and expected to be converted into RFC
 | 2024 goal                                                         | Working towards                       | Owner                               | Teams                       |
 | ----------------------------------------------------------------- | ------------------------------------- | ----------------------------------- | --------------------------- |
 | [Async closures and send bounds][AFE]                             | Async/sync parity                     | [nikomatsakis][], [tmandry][]       | [Lang], [Libs-API]          |
-| [Stabilize key RFL features, RFL on CI][LK] [![Help wanted]][LKH] | Linux builds on stable Rust           | [nikomatsakis][], [Josh-Triplett][] | [Lang], [Libs-API], [Infra] |
+| [Stabilize key RFL features, RFL on CI][LK] [![Help wanted]][LKH] | Linux builds on stable Rust           | [nikomatsakis][], [joshtriplett][]  | [Lang], [Libs-API], [Infra] |
 
 
 ### WIP flagship goals
 
-The following goals were submitted and are sitll "under revision". They are unlikely to become flagship goals this round, but some parts of them may be adopted as team goals.
+The following goals were submitted and are still "under revision". They are unlikely to become flagship goals this round, but some parts of them may be adopted as team goals.
 
 | 2024 goal                                                         | Working towards                       | Owner                               | Teams                       |
 | ----------------------------------------------------------------- | ------------------------------------- | ----------------------------------- | --------------------------- |
 | [Ergonomics initiative: clones and partial borrows][EI]           | Entry-level Rust developer experience | [jkelleyrtp][]                      | [Lang]                      |
-| [Faster iterative builds][FIB]                                    | Entry-level Rust dev experience       | ![WIP][wip]                         | [jkelleyrtp][]              | [Lang], [Compiler]
-| [Rust for Scientific Computing][SCI]                              | Rust for Scientific Computing         | ![WIP][wip]                         | [ZuseZ4][]                  | [Lang], [Compiler]
-| [Towards seamless C support][SCS]                                 |                                       | ![WIP][wip]                         | [Josh-Triplett][]           | [Lang], [Compiler]
+| [Faster iterative builds][FIB] ![WIP][wip]                        | Entry-level Rust dev experience       | [jkelleyrtp][]                      | [Lang], [Compiler]          |
+| [Rust for Scientific Computing][SCI] ![WIP][wip]                  | Rust for Scientific Computing         | [ZuseZ4][]                          | [Lang], [Compiler]          |
+| [Towards seamless C support][SCS] ![WIP][wip]                     |                                       | [joshtriplett][]                    | [Lang], [Compiler]          |
```

Comment:
I've changed these because the table was partially broken, and assumed the expected owners were in the teams column. But I'm not sure if these were intended to be orphaned goals and I changed semantics by fixing the layout. Maybe the owners are indeed WIP and the existing names should have been removed instead?